roles/db-create/tasks/main.yml

```diff
@@ -97,26 +97,36 @@
   when: db_password_secret | length == 0
   tags: db-create
 
-- name: Generate command for password randomization (not 11.2)
+- name: Generate command for password randomization for CDB
   set_fact:
     pwd_gen_cmd: echo -e "$({{ pass_param }})\n$({{ pass_param }})\n$({{ pass_param }})"
   when:
-    - oracle_ver != "11.2.0.4.0"
     - db_password_secret | length == 0
+    - container_db | bool
```

Comment:
nit: if converting to a bool in `group_vars/all.yml`, it's probably safer not to explicitly cast to a bool here, to cause the evaluation to error out if a string somehow sneaks in.